CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 * `shopify/jest/no-vague-titles` ([#93](https://github.com/Shopify/eslint-plugin-shopify/pull/93))
 * `shopify/strict-component-boundaries` ([#98](https://github.com/Shopify/eslint-plugin-shopify/pull/98))
 
+### Changed
+* **Breaking** Moved prettier to be a peerDependency, this avoids the potential for having multiple versions of prettier in the dependency graph ([#107](https://github.com/Shopify/eslint-plugin-shopify/pull/107))
```

Comment:
Can we add a note here for projects to install prettier themselves. Frequent visitors are more likely to read the changelog over readme history.